vaadin-spring/src/main/java/com/vaadin/flow/spring/VaadinServletContextInitializer.java

```diff
@@ -397,12 +412,6 @@ public void contextInitialized(ServletContextEvent event) {
                     || !config.enableDevServer()) {
                 return;
             }
-            Map<String, TaskExecutor> executors = appContext
```

Comment:
Why is this removed?

Reply:
Because it's not needed (not used anywhere)